song/app.js

```diff
@@ -94,6 +98,54 @@ app.post("/content", async (req, res, next) => {
   );
   res.status(201).json({ message: "postCreated" });
 });
+// 좋아요 기능
+app.post("/like", async (req, res, next) => {
+  const { userId, postId } = req.body;
+  await appDataSource.query(
+    `
+    INSERT INTO likes(
+      user_id,
+      post_id
+    ) VALUES (?, ?)
+    `,
+    [userId, postId]
+  );
+
+  res.status(200).json({ message: "likeCreated" });
+});
+
+app.patch("/update", async (req, res, next) => {
```

Comment:
```suggestion
app.patch("/posts", async (req, res) => {
```
1. http method인 `PATCH` 자체가 수정을 의미하기 때문에, 엔드포인트(목적 자원)에 `/update`라고 작성하는 것 보다는, 어떤걸 PATCH 할 지 써주는게 보다 직관적입니다. `/posts`처럼요. 이와 같이 작성하면 http method와 엔드포인트의 조합만으로 무엇을(what), 어떻게(how) 하겠다는 것을 자명하게 나타낼 수 있습니다. 이러한 엔드포인트 설계 아키텍쳐를 REST라고 하는데 이는 금주에 세션에서 자세한 내용과 함께, 다시 한 번 다룰 예정이니 지금은 '그렇구나~' 인지만 하고 넘어가시면 될 것 같습니다. 수정해주세요!
2. 콜백 함수의 세번째 인자로 받는 `next`는 express에서 다음 middleware로 넘기는 next function을 위한 자리인데 지금은 사용하지 않으니 없어도 됩니다.